ts/features/profile/components/UserProfileItem.tsx

```diff
@@ -32,7 +33,7 @@ const styles = StyleSheet.create({
 });
 
 const UserProfileItem = (props: Props): React.ReactElement => (
```

Comment:
```suggestion
const UserProfileItem = (props: WithTestID<Props>): React.ReactElement => (
```
In this way you can avoid
`testID?: string;`